src/plugins/home/public/services/feature_catalogue/feature_catalogue_registry.ts

```diff
@@ -86,6 +89,10 @@ export class FeatureCatalogueRegistry {
     this.capabilities = capabilities;
   }
 
+  /**
+   * @deprecated
```

Comment:
this is still needed, only in `src/plugins/home/public/application/application.tsx`